File: demo-source/乘法口诀表.c
#include <stdio.h>

int main(void) {
    for (int row = 1; row <= 9; row++) {
        for (int column = 1; column <= row; column++) {
            printf("%d×%d=%2d\t", column, row, column * row);
        }
        putchar('\n');
    }
    return 0;
}

File: demo-source/乘法口诀表.pl
use strict;
use warnings;
use utf8;
binmode STDOUT, ':encoding(UTF-8)';

for my $row (1 .. 9) {
    for my $column (1 .. $row) {
        printf "%d×%d=%2d\t", $column, $row, $column * $row;
    }
    print "\n";
}
